journalpump/journalpump.py

```diff
@@ -661,11 +662,12 @@ def read_single_message(self, reader):
             return False
         except Exception as ex:  # pylint: disable=broad-except
             self.log.exception("Unexpected exception while handling entry for %s", reader.name)
-            self.stats.unexpected_exception(ex=ex, where="mainloop", tags=self.make_tags({"app": "journalpump"}))
+            if self.stats is not None:
+                self.stats.unexpected_exception(ex=ex, where="mainloop", tags=self.make_tags({"app": "journalpump"}))
             time.sleep(0.5)
             return False
 
-    def read_all_available_messages(self, reader, hits):
+    def read_all_available_messages(self, reader, hits: MutableMapping[str, Any]):
```

Comment:
Perhaps this is a personal opinion, but I believe in all or nothing when it comes to types in method signatures

Reply:
There are commented out lines in `mypy.ini` which represent the settings I wanted to enable. But the practical reality of that was some of them imply large code refactors. The initial goal is to get mypy and the typechecker running and passing, so they can later be tightened (which again: requires larger code refactors to do soundly, but makes the PR even bigger).